README.md

```diff
@@ -210,6 +229,23 @@ module MyBot
 end
 ```
 
+Hooks can also be written as blocks inside the `SlackBotRuby::Server` class, for example
+
+```ruby
+module MyBot
+  class MyServer < SlackRubyBot::Server
+    hook 'hello' do |client, data|
```

Comment:
I kept it explicit like the commands, I get that it's an event, but `on` feels way to short and generic. 

Up to you, happy to comply :)